cmd/switch.go

```diff
@@ -15,55 +15,103 @@
 package cmd
 
 import (
+	"fmt"
+
 	"github.com/spf13/cobra"
 
 	"github.com/tetratelabs/getistio/api"
 	"github.com/tetratelabs/getistio/src/istioctl"
 	"github.com/tetratelabs/getistio/src/util/logger"
 )
 
+type switchFlags struct {
+	name, version, flavor string
+	flavorVersion         int64
+}
+
 func newSwitchCmd(homedir string) *cobra.Command {
-	var (
-		flagVersion       string
-		flagFlavor        string
-		flagFlavorVersion int
-	)
+	var flag switchFlags
 
 	cmd := &cobra.Command{
-		Use:   "switch <istio version>",
+		Use:   "switch <istio version | flavor | flavor-version>|<istio version full name>",
 		Short: "Switch the active istioctl to a specified version",
 		Long:  `Switch the active istioctl to a specified version`,
 		Example: `# switch the active istioctl version to version=1.7.4, flavor=tetrate and flavor-version=1
-$ getistio switch --version 1.7.4 --flavor tetrate --flavor-version=1`,
+$ getistio switch --version 1.7.4 --flavor tetrate --flavor-version=1, 
+$ getistio switch --name 1.7.4-tetrate-v1
+switch also supports to change only one version|flavor|flavorVersion flag and follow the rest settings in active version`,
```

Comment:
please follow the same style. https://github.com/tetratelabs/getistio/blob/main/cmd/fetch.go#L38-L57